includes/footer.php

```diff
@@ -74,11 +73,11 @@
                         <span class="sr-only">Pause/Resume</span>
                     </a>
 
-                    <a class="left carousel-control" href="#" role="button" data-slide="prev" onclick="$('#carouselSponsor').carousel('prev');">
+                    <a class="left carousel-control" href="#carouselSponsor" role="button" data-slide="prev" onclick="$('#carouselSponsor').carousel('prev');">
```

Comment:
The shadow on the arrow buttons looks weird on the logos. Maybe add some CSS to remove it **for this carousel only**? Either inline or using a style sheet is fine (see [`/style/README.md`](https://github.com/FRC-1294/1294-Website/blob/master/style/README.md), noting the time based anti-caching procedure (add date to file name, change file name in `/includes/lower_header.php`).